app/src/main/java/org/luxoft/sdl_core/JavaToNativeBleAdapter.java

```diff
@@ -0,0 +1,58 @@
+package org.luxoft.sdl_core;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+
+public class JavaToNativeBleAdapter extends Thread {
+    public static final String TAG = JavaToNativeBleAdapter.class.getSimpleName();
+    final int WRITE_ID = 1;
+    final int READ_ID = 2;
```

Comment:
```suggestion
    private static final int WRITE_ID = 1;
    private static final int READ_ID = 2;
```

Reply:
Done in 8effe4942498195cce7a335540df658a92a3e296